pulpcore/cli/python/distribution.py

```diff
@@ -53,6 +54,11 @@ def distribution(ctx: click.Context, pulp_ctx: PulpContext, distribution_type: s
     click.option("--base-path"),
     click.option("--publication"),
     repository_option,
+    pulp_option(
+        "--allow-uploads/--block-uploads",
```

Comment:
Really not sure here: Would "--no-allow-uploads" be more intuitive?

Reply:
I know, we also have "--dry-run/--force"...

Reply:
I think I favor the allow/block verbiage, from a user's standpoint.